data/transactions/logic/eval.go

```diff
@@ -1385,6 +1519,19 @@ func opBytesMul(cx *EvalContext) {
 	opBytesBinOp(cx, result, result.Mul)
 }
 
+func opBytesSqrt(cx *EvalContext) {
+	last := len(cx.stack) - 1
+
+	if len(cx.stack[last].Bytes) > MaxByteMathSize {
+		cx.err = errors.New("math attempted on large byte-array")
+		return
+	}
+
+	val := new(big.Int).SetBytes(cx.stack[last].Bytes)
```

Comment:
Setting bytes from the stack could result in a negative value that would panic when Sort is called.

Reply:
SetBytes always treats the bytes as an unsigned integer.
> SetBytes interprets buf as the bytes of a big-endian unsigned integer, sets z to that value, and returns z.